compiler/src/dotty/tools/dotc/inlines/Inliner.scala

```diff
@@ -270,12 +270,15 @@ class Inliner(val call: tpd.Tree)(using Context):
         assert(argss.isEmpty)
         true
 
+  /** The number of enclosing classes of this class, plus one */
+  private def nestingLevel(cls: Symbol) = cls.ownersIterator.count(_.isClass)
```

Comment:
maybe call it outerLevel to avoid confusion with ctx.nestingLevel used in level-checking?

Reply:
I renamed it to `classNestingLevel`.